build_tools/cmake/iree_copts.cmake

```diff
@@ -197,6 +197,10 @@ iree_select_compiler_opts(IREE_DEFAULT_COPTS
     "-Wthread-safety-beta"
     "-Wunused-comparison"
     "-Wvla"
+    "-Wpointer-arith"
```

Comment:
nit: the rest of this list is alphabetized

Reply:
done